TESTS/mbedmicro-rtos-mbed/malloc/main.cpp

```diff
@@ -7,7 +7,11 @@
 #endif
 
 #define NUM_THREADS         5
+#if defined(__CORTEX_A9)
+#define THREAD_STACK_SIZE   DEFAULT_STACK_SIZE
```

Comment:
@0xc0170 

Cortex-A in general will require more stack space than Cortex-M.
Therefore it is necessary to divide the definition by "#ifdef" as same as other test. By using its value CI test sequece is carried out.
For example:
TESTS\mbedmicro-rtos-mbed\basic\main.cpp

In Cortex-A, DEFAULT_STACK_SIZE is 2048 byte. Maybe in Cortex-M, it is 256Byte.